internal/cmd/cmd_del.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/dicedb/dice/wire"
+)
+
+var cDEL = &DiceDBCommand{
+	Name:      "DEL",
+	HelpShort: "DEL deletes all the specified keys in args list",
+	Eval:      evalDEL,
+}
+
+func init() {
+	commandRegistry.AddCommand(cDEL)
+}
+
+// evalDEL deletes all the specified keys in args list.
+//
+// Parameters:
+//   - c *Cmd: The command context containing the arguments
+//   - s *dstore.Store: The data store instance
+//
+// Returns:
+//   - *CmdRes: Response containing the count of total deleted keys
+//   - error: Error if wrong number of arguments or wrong value type
+func evalDEL(c *Cmd, s *dstore.Store) (*CmdRes, error) {
+	if len(c.C.Args) < 1 {
+		return cmdResNil, errWrongArgumentCount("DEL")
+	}
+
+	var count int64
```

Comment:
could be an int32.

Reply:
Understood.

Reply:
@arpitbbhayani `wire.Response_VInt` supports int64, we can either:
1. convert int32 to int64 via int64(count)
2. add support for int32 in cmd.proto (wire.Response_VInt32 and Response_VInt64)

Reply:
You don't need that support. The value of count does not depend on anything in the proto. You will never have massive number of keys being deleted. Change this to `int`

Reply:
Updated.